File: app/src/main/java/com/example/helloworld/MainActivity.java
package com.example.helloworld;

import android.content.Intent;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import com.google.android.material.floatingactionbutton.FloatingActionButton;
import com.google.android.material.snackbar.Snackbar;


public class MainActivity extends AppCompatActivity {

    private Button quiz;
    private Button chart;
    private Button mBtLaunchActivity;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
        quiz = findViewById(R.id.launch_quiz_activity);
        chart = findViewById(R.id.launch_chart_activity);
        mBtLaunchActivity = findViewById(R.id.bt_launch_activity);

        mBtLaunchActivity.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                launchBMIActivity();
            }
        });

        quiz.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                launchQuizActivity();
            }
        });
        chart.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                launchChartActivity();
            }
        });
        FloatingActionButton fab = findViewById(R.id.fab);
        fab.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
                        .setAction("Action", null).show();
            }
        });
    }

    private void launchBMIActivity() {

        Intent intent = new Intent(this, ProfileActivity.class);
        startActivity(intent);
    }

    private void launchQuizActivity() {

        Intent intent = new Intent(this, QuizActivity.class);
        startActivity(intent);
    }
    private void launchChartActivity() {

        Intent intent = new Intent(this, ChartActivity.class);
        startActivity(intent);
    }
    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        // Inflate the menu; this adds items to the action bar if it is present.
        getMenuInflater().inflate(R.menu.menu_main, menu);
        return true;
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        // Handle action bar item clicks here. The action bar will
        // automatically handle clicks on the Home/Up button, so long
        // as you specify a parent activity in AndroidManifest.xml.
        int id = item.getItemId();

        //noinspection SimplifiableIfStatement
        if (id == R.id.action_settings) {
            return true;
        }

        return super.onOptionsItemSelected(item);
    }
}

File: KotlinStart/app/src/main/java/com/example/kotlinstart/MainActivity.kt
package com.example.kotlinstart

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById<Button>(R.id.chart).setOnClickListener(View.OnClickListener {
            launchChartActivity()
        })
        findViewById<Button>(R.id.bmi).setOnClickListener(View.OnClickListener {
            launchBMIActivity()
        })
        findViewById<Button>(R.id.quiz).setOnClickListener(View.OnClickListener {
            launchQuizActivity()
        })
    }
    fun launchChartActivity(){
        val intent = Intent(this, ChartActivity::class.java)
        startActivity(intent)
    }
    fun launchBMIActivity(){
        val intent = Intent(this, BmiActivity::class.java)
        startActivity(intent)
    }
    fun launchQuizActivity(){
        val intent = Intent(this, QuizActivity::class.java)
        startActivity(intent)
    }

}
